Plan/src/main/java/com/djrapitops/plan/system/tasks/velocity/EnableConnectionTask.java

```diff
@@ -0,0 +1,21 @@
+package com.djrapitops.plan.system.tasks.velocity;
+
+import com.djrapitops.plan.system.info.connection.ConnectionSystem;
+import com.djrapitops.plan.system.info.connection.WebExceptionLogger;
+import com.djrapitops.plan.system.info.request.GenerateNetworkPageContentRequest;
+import com.djrapitops.plugin.task.AbsRunnable;
+
+public class EnableConnectionTask extends AbsRunnable {
```

Comment:
Duplication of this class is unnecessary, as only benefit is having it in velocity specific package.
This can be sorted later by refactoring common classes into network specific names